src/components/forms/AddServiceForm.js

```diff
@@ -171,8 +174,13 @@ function AddServiceForm({
           >
             <Select size="large" placeholder="Select Recipient">
               {recipients.map(recipient => (
-                <Select.Option key={recipient.id} value={recipient.id}>
-                  {recipient.first_name + ' ' + recipient.last_name}
+                <Select.Option
```

Comment:
Please see my comments above about changing fields on objects. Make sure you QA this really well.